arcade/__init__.py

```diff
@@ -294,6 +294,11 @@ def configure_logging(level: int = None):
 from .sprite_list import get_sprites_at_exact_point
 from .sprite_list import get_sprites_at_point
 
+from .backgrounds import BackgroundTexture
+from .backgrounds import Background
+from .backgrounds import BackgroundGroup
+from .backgrounds import ParallaxGroup
```

Comment:
I think we should stop adding things to the arcade namespace. It's getting way too cluttered. Users having to import from the `arcade.backgrounds` module is fine.